scratch

```diff
@@ -0,0 +1,105 @@
+{
```

Comment:
What is this file? Accidentally added?

Reply:
Yes accident thanks! (sample of the json you get out of the alma full dump event)